drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -123,9 +123,9 @@ def self.configuration
 
     def external_ids
       object.external_identifiers.
-        map do |key, vals|
+        map do |vals|
           {
-            id: [key, object.id].join(':'),
+            id: [vals[:type], object.id].join(':'),
```

Comment:
This is just to get a unique `id` for graphql, right? Maybe it would be simpler to use the value itself to ensure the uniqueness. That will makes things easier in the future if we introduce other IDs that may have multiple values per client.


It would also allow us to resolve the `type` on the `Types::HmisSchema::ExternalIdentifier` as an Enum. That would be helpful because in the frontend right now we're using the label ("MCI ID") to identify the id. No need to make that change, but I think it's another reason to have standardized `type`s per External ID.

```suggestion
            id: [vals[:type], vals[:id].join(':'),
```

and then on line 196
```
type: :mci_id,
```

Reply:
Yes, it's just to have a unique ID. Agreed on the`type` enum, I went ahead and added that and updated the labels in the IDs table in the client profile in the frontend to use the `type` with `HmisEnum` rather than `label`. I also see that we're using external identifiers in referrals, but I'm not familiar with that part of the application so I left the `label` field on the `ExternalIdentifier` type for now. We could remove it in favor of `type` at some point.